systemtest/src/test/java/io/strimzi/systemtest/KafkaST.java

```diff
@@ -99,16 +99,29 @@
     @OpenShiftOnly
     void testDeployKafkaClusterViaTemplate() {
         createCustomResources("../examples/templates/cluster-operator");
+        String appName = "strimzi-ephemeral";
         Oc oc = (Oc) KUBE_CLIENT;
         String clusterName = "openshift-my-cluster";
-        oc.newApp("strimzi-ephemeral", map("CLUSTER_NAME", clusterName));
+        oc.newApp(appName, map("CLUSTER_NAME", clusterName));
         oc.waitForStatefulSet(zookeeperClusterName(clusterName), 3);
         oc.waitForStatefulSet(kafkaClusterName(clusterName), 3);
         oc.waitForDeployment(entityOperatorDeploymentName(clusterName), 1);
 
         //Testing docker images
         testDockerImagesForKafkaCluster(clusterName, 3, 3, false);
 
+        //Testing labels
+        verifyLabelsOnCOPod(NAMESPACE);
```

Comment:
These all methods could be called from one method which will contains all of them. Test itself will be much cleaner I think.